motor/discovery/aws/ecs.go

```diff
@@ -238,11 +257,19 @@ func ecsContainerToAsset(ctx context.Context, account string, region string, c e
 			asset.Labels[key] = value
 		}
 	}
-	asset.Labels["cluster_name"] = clusterName
-	asset.Labels["task_definition_arn"] = *task.TaskDefinitionArn
+	asset.Labels[ClusterNameLabel] = clusterName
+	asset.Labels[TaskDefinitionArnLabel] = *task.TaskDefinitionArn
 	return asset
 }
 
+const (
+	ContainerName          = "container_name"
+	ClusterNameLabel       = "cluster_name"
+	TaskDefinitionArnLabel = "task_definition_arn"
+	TaskId                 = "task_id"
```

Comment:
Didn't see where ContainerName and TaskId are used

Reply:
they will be used in the lambda/at ecs exec scan time